frontend/src/relay/auth.tsx

```diff
@@ -102,14 +102,14 @@ export const getJwt = (service: JwtService, event?: string): Promise<string> =>
         let gotResult = false;
         let out: authJwtQuery["response"];
         const query = graphql`
-            query authJwtQuery($service: JwtService!, $event: ID) {
-                jwt(service: $service, event: $event)
+            query authJwtQuery($service: JwtService!, $event: ID, $ocId: String) {
+                jwt(service: $service, event: $event, opencastId: $ocId)
             }
         `;
         fetchQuery<authJwtQuery>(
             environment,
             query,
-            { service, event },
+            { service, event, ocId: event },
```

Comment:
Mh it's not amazing passing `event` as two different fields. I get it, the `$event` var in gql is an `ID` which is not true for the Opencast ID... maybe the `getJwt` function should be changed then? Replace the second argument `event?` with `payload: { event?: string; opencastId?: string }` maybe?